elibrary-core/source/src/controller/java/com/javafee/elibrary/core/tabbedform/BookAddModEvent.java

```diff
@@ -114,7 +112,7 @@ private void modificateBook() {
 				HibernateUtil.beginTransaction();
 				HibernateUtil.getSession()
 						.evict(bookTableModel.getBook((Integer) Params.getInstance().get("selectedRowIndex")));
-				HibernateUtil.getSession().update(Book.class.getName(), bookShallowClone);
+				HibernateUtil.getSession().saveOrUpdate(Book.class.getName(), bookShallowClone);
```

Comment:
'After saving book data authors/categories/publishing houses were not properly saved.' (from tasks comments)